src/app/cart/_components/cart-list-item.tsx

```diff
@@ -0,0 +1,139 @@
+import { Checkbox } from "@/components/ui/checkbox";
+import { Chip } from "@/components/ui/chip";
+import { IconButton } from "@/components/ui/icon-button";
+import { cn } from "@/lib/utils";
+import {
+  CartProduct,
+  MAX_CART_ITEM_QUANTITY,
+  useDeleteCartItem,
+  useUpdateCartItem,
+} from "@/services/cart";
+import { calculateCouponAppliedPrice } from "@/services/coupon";
+import { CartItemId } from "@/services/shared";
+import { MinusIcon, PlusIcon, XIcon } from "lucide-react";
+import { usePageContext } from "../_context";
+
+type CartListItemProps = {
+  id: CartItemId;
+  quantity: number;
+  product: CartProduct;
+};
+
+export const CartListItem = ({ id, quantity, product }: CartListItemProps) => {
+  const { mutate: updateCartItem } = useUpdateCartItem();
+  const { mutate: deleteCartItem } = useDeleteCartItem();
+
+  const { selectedCoupon, selectedProductIds, toggleSelectedProductId, removeSelectedProductId } =
+    usePageContext();
+
+  const isProductSelected = selectedProductIds.some(
+    (selectedProductId) => selectedProductId === product.id,
+  );
+
+  const onMinusButtonClick = () => {
+    updateCartItem({
+      id: id,
+      body: {
+        quantity: quantity - 1,
+      },
+    });
```

Comment:
shorthand 문법 사용해주시면 좋을 것 같네요!

```suggestion
    updateCartItem({
      id,
      body: {
        quantity: quantity - 1,
      },
    });
```

Reply:
요거 quantity를 이렇게 넘겨주는 방식보다 decrease, increase 쌍이 별도로 있는게 더 좋을 것 같아요.